packages/manager/modules/telecom-spare/src/constant.js

```diff
@@ -0,0 +1,7 @@
+const SPARE_BRAND = {
```

Comment:
`SPARE_BRAND` seems only used in `packages/manager/modules/telecom-spare/src/filters/formatModemBrand/format-modem-brand.filter.js` so the constant file should move in `packages/manager/modules/telecom-spare/src/filters/formatModemBrand/`;